scikeras/wrappers.py

```diff
@@ -35,7 +35,11 @@
 from scikeras.utils.transformers import ClassifierLabelEncoder, RegressorTargetEncoder
 
 
-_kwarg_warn = "Passing `**kwargs` to `{0}` is not fully supported by the sklearn API. Instead, set arguments at initialization (`BaseWrapper({0}__{1}={2})`) or use `set_params`"
+_kwarg_warn = """Passing `**kwargs` to `{0}` is not fully supported by the sklearn API.
+See our docs for more details: https://www.adriangb.com/scikeras/refs/heads/master/migration.html#variable-keyword-arguments-in-fit-and-predict
+The following `**kwargs` were used:
```

Comment:
```suggestion
The following parameters were passed to `{0}`:
```

Reply:
And shouldn't the error message provide a method to resolve the warning?

Reply:
It links to the docs, which I think provide a more detailed explanation and avenues for resolution than is possible in a warning. Do you feel that linking to the docs does not accomplish this?

Reply:
Yes, linking to the docs provides an example. Yes, users can look at the docs and that will resolve their issue. In fact, you *should* link to the full example where you resolve the issue.

I'm not asking to remove the link. I think an error message should say "to resolve this issue, [method]. For more detail, see [link]."

I think this makes SciKeras easier to use because it requires less work to get the code to run. What if I'm developing without internet access? I've often done that on airplanes or road trips.

Reply:
That's fair. Do you think we can say `put the parameters in the constructor or use set_params` or something simple like that, without injecting parameter-value examples?

Reply:
Why the hesitation to include parameter-value examples? I suppose you could use an illustrative example like `foo="bar"`.

Reply:
> I suppose you could use an illustrative example like foo="bar"

The warning now has an illustrative example like you suggest above.